enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -0,0 +1,35 @@
+"""
+management command for assigning existing enterprise users to their respective enterprise roles.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from enterprise.constants import ENTERPRISE_ADMIN_ROLE, ENTERPRISE_DATA_API_ACCESS_GROUP, ENTERPRISE_LEARNER_ROLE
+from enterprise.models import EnterpriseCustomerUser, SystemWideEnterpriseRole, SystemWideEnterpriseUserRoleAssignment
+
+
+class Command(BaseCommand):
+    """ management command class """
+    help = 'Assigns existing enterprise users to their specific roles.'
+
+    def handle(self, *args, **options):
+        enterprise_admin_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_ADMIN_ROLE)
+        enterprise_learner_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_LEARNER_ROLE)
+
+        # Assign admin role to users with enterprise data api access.
+        users_with_enterprise_access = User.objects.filter(groups__name=ENTERPRISE_DATA_API_ACCESS_GROUP)
+        for user_with_enterprise_access in users_with_enterprise_access:
```

Comment:
we should only be doing this for users who are not staff users. i am thinking that we should introduce an edx_enterprise_operator system wide role to apply to the staff users who need it, but you don't need to worry about that right now in the scope of this PR.